src/containers/pages/IRS/JurisdictionsReport/tests/namibia_configs.test.tsx

```diff
@@ -0,0 +1,123 @@
+reducerRegistry.register(IRSPlansReducerName, IRSPlansReducer);
+reducerRegistry.register(GenericJurisdictionsReducerName, GenericJurisdictionsReducer);
+
+/** set up data in the store needed for this view */
+const focusAreaData = superset.processData(fixtures.ZambiaJurisdictionsJSON) || [];
+const jurisdictionData = superset.processData(fixtures.ZambiaJurisdictionsJSON) || [];
+
+store.dispatch(fetchGenericJurisdictions('nm-jurisdictions', jurisdictionData));
+store.dispatch(fetchGenericJurisdictions('nm-focusAreas', focusAreaData));
+
+const history = createBrowserHistory();
+
+describe('components/IRS Reports/JurisdictionReport', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('renders correctly', async () => {
+    fetch.mockResponseOnce(JSON.stringify({}));
+    const mock: any = jest.fn();
+
+    const supersetServiceMock: any = jest.fn();
+    supersetServiceMock.mockImplementation(async () => []);
+
+    let jurisdictions =
+      getGenericJurisdictionsArray(
+        store.getState(),
+        'nm-jurisdictions',
+        '9f1e0cfa-5313-49ff-af2c-f7dbf4fbdb9d'
+      ) || [];
+    jurisdictions = jurisdictions.concat(
+      getGenericJurisdictionsArray(
+        store.getState(),
+        'nm-focusAreas',
+        '9f1e0cfa-5313-49ff-af2c-f7dbf4fbdb9d'
+      )
+    );
+
+    const props = {
+      history,
+      jurisdictions,
+      location: mock,
+      match: {
+        isExact: true,
+        params: {
+          planId: (plans[0] as IRSPlan).plan_id,
+        },
+        path: `${REPORT_IRS_PLAN_URL}/:planId`,
+        url: `${REPORT_IRS_PLAN_URL}/${(plans[0] as IRSPlan).plan_id}`,
+      },
+      plan: plans[0] as IRSPlan,
+      service: supersetServiceMock,
+    };
+
+    const wrapper = mount(
+      <Router history={history}>
+        <JurisdictionReport {...props} />
+      </Router>
+    );
+    const helmet = Helmet.peek();
+    await flushPromises();
+    expect(toJson(wrapper.find('BreadcrumbItem li'))).toMatchSnapshot('breadcrumbs');
+    expect(toJson(wrapper.find('h3.page-title'))).toMatchSnapshot('page title');
+    expect(helmet.title).toEqual('IRS Reporting: IRS 2019-09-05 TEST');
+    expect(wrapper.find('DrillDownTable').props()).toMatchSnapshot({
+      columns: expect.any(Object) /** just for purposes of making snapshot smaller */,
+      data: expect.any(Object) /** just for purposes of making snapshot smaller */,
+    });
+    expect(wrapper.find('DrillDownTable').props().data).toEqual(jurisdictions);
+    expect((wrapper.find('DrillDownTable').props() as any).columns).toEqual(
+      IRSTableColumns.namibia2019
+    );
+    expect(wrapper.find('IRSIndicatorLegend').length).toEqual(1);
+    expect(wrapper.find('IRSIndicatorLegend').props()).toEqual({
+      indicatorRows: 'namibia2019',
+      indicatorThresholds: indicatorThresholdsIRS,
+      indicatorThresholdsLookUp: indicatorThresholdsLookUpIRS,
+    });
+    expect(supersetServiceMock).toHaveBeenCalledTimes(3);
```

Comment:
Please also test the content of the calls e.g. https://github.com/onaio/reveal-frontend/blob/master/src/containers/pages/IRS/JurisdictionsReport/tests/index.test.tsx#L150

Reply:
Tests added